matrices.tex

```diff
@@ -25,6 +25,8 @@
 {x - sqrt(y*(x + y)): 1, x + sqrt(y*(x + y)): 1}
 \end{verbatim}
 
+%% TODO a Python list is not a dense representation...
+%% Is this supposed to say flat? - scopatz
```

Comment:
I understand that it is a dense matrix, but the list isn't a dense data structure. A list of lists of symbols would still be a dense matrix but the representation would not be any less dense as a data structure (in Python). I think this could be reworded because it implies that the flatness of the list makes the matrix dense, which is not true.